packages/core/src/commands/insertContentAt.ts

```diff
@@ -67,13 +73,15 @@ export const insertContentAt: RawCommands['insertContentAt'] = (position, value,
       updateSelection: true,
       applyInputRules: false,
       applyPasteRules: false,
+      slice: true,
       ...options,
     }
 
     let content: Fragment | ProseMirrorNode
 
     try {
       content = createNodeFromContent(value, editor.schema, {
+        slice: options.slice,
```

Comment:
I am concerned this is a breaking change. Before these changes, the `createNodeFromContent` was called with the `slice` value being false. But now it is called with a default value of `true`. If the default value of options.slice was `false` then this would not be a breaking change. But if the value should be true by default, then you have my approval. Just wanted to point this out.

Reply:
I would also rather keep the previous default to not cause breaking changes.